appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java

```diff
@@ -1018,9 +1019,9 @@ private void setTheme(String theme) {
     }
     if (theme.equals("AppTheme")) {
       blackIcons = false;
-      formWidget.setStylePrimaryName("ode-SimpleMockFormDark");
+      formWidget.addStyleDependentName("ode-SimpleMockFormDark");
```

Comment:
I'm not sure this accomplishes what we want but I will need to test it to be sure.